docs/shared-content-source/docs/modules/ROOT/partials/include.adoc

```diff
@@ -1,4 +1,34 @@
-// global attributes
-:page-partial:
 
-:cli-plugin: kubectl
+// global attributes if you update, make sure to update in docs/modules/RoOT:partials.adoc too
+:supported-java-v: >= 8
+:supported-sbt-v: version 1.2.8 or higher
+:supported-docker-v: version 18.09 or higher
+:supported-helm-v: version 3 or higher
+
+:kubernetes-platform-name:     Kubernetes
+:kubernetes-platform-version:  1.12
+:kubectl-plugin-version:       KUBECTL_PLUGIN_VERSION
+
+:cli:                          kubectl
+:cli-plugin:                   kubectl
+
+:latest-openshift-platform-version:        3.11
+:minimum-kubernetes-on-openshift-version:  1.11
+:minimum-openshift-platform-version:       3.11
+:minimum-platform-version-major-dot-minor: 3.11
+:minimum-sbt-version:                      0.13.7
+:latest-sbt-version: 					   1.3.8	
```

Comment:
The latest sbt 1.3.X is 1.3.13 but we are likely to support 1.4.X anytime soon after this get merged:
https://github.com/lightbend/cloudflow/pull/880

Reply:
I'll update to 1.3.13, pls let me know when we support 1.4.X

Reply:
since, well, next version :-)